tests/get_job_test.py

```diff
@@ -0,0 +1,23 @@
+from c20_server.get_job import app
+
+response = app.test_client().get('/')
```

Comment:
This code needs to be in a test

Reply:
Does this import need to be in a test as well?

Reply:
This has been put into a test and a fixture has been added as well.